qiskit/circuit/library/standard_gates/__init__.py

```diff
@@ -11,7 +11,76 @@
 # that they have been altered from the originals.
 
 """
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
+>>>>>>> 57939ba279d95b5e3f56a08e186f5e1459c3c44f
+=============================================================
+Standard gates (:mod:`qiskit.circuit.library.standard_gates`)
+=============================================================
+
+.. autosummary::
+   :toctree: ../stubs/
+
+   C3XGate
+   C3SXGate
+   C4XGate
+   CCXGate
+   DCXGate
+   CHGate
+   CPhaseGate
+   CRXGate
+   CRYGate
+   CRZGate
+   CSwapGate
+   CSXGate
+   CUGate
+   CU1Gate
+   CU3Gate
+   CXGate
+   CYGate
+   CZGate
+   CCZGate
+   HGate
+   IGate
+   MCPhaseGate
+   PhaseGate
+   RCCXGate
+   RC3XGate
+   RXGate
+   RXXGate
+   RYGate
+   RYYGate
+   RZGate
+   RZZGate
+   RZXGate
+   XXMinusYYGate
+   XXPlusYYGate
+   ECRGate
+   SGate
+   SdgGate
+   CSGate
+   CSdgGate
+   SwapGate
+   iSwapGate
+   SXGate
+   SXdgGate
+   TGate
+   TdgGate
+   UGate
+   U1Gate
+   U2Gate
+   U3Gate
+   XGate
+   YGate
+   ZGate
+   GlobalPhaseGate
```

Comment:
I think you've accidentally added these are part of trying to resolve a conflicted merge.  If you look at at the "Review" tab at the top of the GitHub PR, you can see the actual changeset of your PR (additions start with a `+` and are green, removals start with a `-` and are red).  If you could restore this docstring back to just the white lines (unchanged), that'd be great, thanks.

The `GlobalPhaseGate` line you've added actually needs adding in `qiskit/circuit/library/__init__.py` for it to appear in the documentation.

Reply:
Thank you for letting me know! Removed it.